packages/swapper/tsconfig.json

```diff
@@ -17,6 +17,9 @@
     },
     {
       "path": "../caip",
+    },
+    {
+      "path": "../chain-adapters",
```

Comment:
we should add the corresponding workspace dependency to package.json. looks like `types` and `utils` aren't quite aligned either.

Reply:
588221b07f feelsgoodman.jpg